package.json

```diff
@@ -466,9 +466,15 @@
     },
     {
       "author": "whyrusleeping",
-      "hash": "QmQgLZP9haZheimMHqqAjJh2LhRmNfEoZDfbtkpeMhi9xK",
+      "hash": "QmWRCn8vruNAzHx8i6SAXinuheRitKEGu8c7m26stKvsYx",
       "name": "go-testutil",
-      "version": "1.1.12"
+      "version": "1.1.11"
```

Comment:
Is go-testutil being downgraded?

Reply:
hrm... thats probably the fault of a rebase gone rogue. I'll take care of it.

Reply:
Ah, its because we accidentally updated go-testutil in master without bubbling the changes from the rest of the deps at the same time. I think this change is safe for now, we can bubble up those deps and fix things later.